### PR TITLE
Update login

### LINKED ISSUE #228: Updating the  on-boarding
We are updating the onboarding screens. 

1_Adding color backgrounds
2_Change first texts, adding a hyperlink to faq 

_Design material: in **Yellow color**_ https://www.figma.com/file/ZTqF54s6CSHTwAxp0U0JjQ/circles-myxogastria?node-id=3820%3A19832

### LINKED ISSUE #262: Change account creation process
We will implement a new Ux and UI design of our creating Wallet flow. 
Includes:

- New process dived in 3 steps
- New visuals
- Reactive design, name and picture appear in place when people add them. 

**Design Material**
_Marked in red_  https://www.figma.com/file/ZTqF54s6CSHTwAxp0U0JjQ/Circles-myxogastria?node-id=4350%3A18052

### LINKED ISSUE #315: Update login screen
https://www.figma.com/file/ZTqF54s6CSHTwAxp0U0JjQ/Circles-myxogastria?node-id=4350%3A34243

### PR DESCRIPTION
**Pull request by mikozet**
Closes #315

### CODE REVIEW

**Comment by mikozet** [comment]
This PR contains work for two tasks:
https://github.com/CirclesUBI/circles-myxogastria/issues/315
as well as
https://github.com/CirclesUBI/circles-myxogastria/issues/262

File: src/hooks/scrollTrigger.js [hunk 1]
```diff
@@ -0,0 +1,4 @@
+import { useScrollTrigger } from '@material-ui/core';
+
+export const useCustomScrollTrigger = () =>
+  useScrollTrigger({ disableHysteresis: true, threshold: 0 });
```
Comment (louilinn) [inline src/hooks/scrollTrigger.js:4]: What does this change? I tried to understand from search but didn'tgo so well.. :sweat_smile:

File: src/views/Onboarding.js [hunk 2]
```diff
@@ -25,12 +25,65 @@ import notify, { NotificationsTypes } from '~/store/notifications/actions';
 import { createNewAccount } from '~/store/onboarding/actions';
 import logError, { formatErrorMessage } from '~/utils/debug';
 
-const useStyles = makeStyles(() => ({
+const moveUpFront = (theme) => ({
```
Comment (louilinn) [inline src/views/Onboarding.js:28]: how does this work?

**Review by louilinn — APPROVED** [review]
Looks good! I reviewed the code but havent tested the UI, let's do that in preview with Lucia :)

**Pull request merged**